packages/pwa-kit-extension-sdk/src/configs/webpack/application-extensions-loader.ts

```diff
@@ -56,12 +58,16 @@ export const ruleForApplicationExtensibility = (options: any = {}) => {
     const {loaderOptions = {}} = options
     const {target = DEFAULT_TARGET, configured} =
         loaderOptions as ApplicationExtensionsLoaderOptions
+    const testRegExp = new RegExp(
+        `${target === 'node' ? 'express' : 'react'}/placeholders/application-extensions.js`,
+        'i'
+    )
 
     return {
-        test: new RegExp(
-            `${target === 'node' ? 'express' : 'react'}/placeholders/application-extensions.js`,
-            'i'
-        ),
+        test: (source: string) => {
+            const posixPath = path.posix.join(...source.split(path.sep))
+            return testRegExp.test(posixPath)
+        },
```

Comment:
Fixing tests on windows.. here we ensure we only test against posix paths by first normalizing the path and testing it.